src/sortStation.c

```diff
@@ -0,0 +1,78 @@
+#include "stack.h"
+#include <stdbool.h>
+#include <stdio.h>
+#include <string.h>
+
+// Функция для расставоения приоритетов операций
+int priority(char sim)
+{
+    if ((sim == '+') || (sim == '-')) {
+        return 1;
+    }
+    if ((sim == '*') || (sim == '/')) {
+        return 2;
+    }
+    return 0;
+}
+
+// Проверяет операнд или нет
+bool check(char sim)
```

Comment:
```suggestion
bool isOperand(char sim)
```

(Поясняющего комментария не достаоточно, так как код вызывающий функцию check его не видит)